Procfile

```diff
@@ -1 +1 @@
-web: npm start
+web: npm run migrate_pg && npm start -- $NETWORK
```

Comment:
is pre-migrating necessary? migration is handled inside AN now during runtime

Reply:
This is probably no longer necessary, indeed. I'll remove it.